jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -21,6 +21,17 @@
  */
 package com.uber.jaeger;
 
+import java.io.InputStream;
```

Comment:
@vprithvi I have imported https://raw.githubusercontent.com/google/styleguide/gh-pages/intellij-java-google-style.xml and run optimize imports on the whole project. It produces a massive diff https://paste.fedoraproject.org/paste/ZSJs8G3jE~VExhSMKHwNuF5M1UNdIGYhyRLivL9gydE=.

It is inconvenient to go through 22 files and change that manually.